### PR TITLE
Add rtl style rule

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: less/tabzilla.less [hunk 1]
```diff
@@ -31,3 +31,8 @@
         z-index: 1;
     }
 }
+.html-rtl {
+    #tabzilla {
```
Comment (alexgibson) [inline less/tabzilla.less]: Nit: fix indentation

**Comment by alexgibson** [comment]
Worth adding a very short note to the readme on how to use the rtl styling?

**Comment by alexgibson** [comment]
r+wc :sake:

**Comment by schalkneethling** [comment]
Yeah, good point. I will add a little something.

**Comment by alexgibson** [comment]
Could we not use the global `dir` attribute too perhaps, instead of a custom CSS class name? Just a thought.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes

**Comment by schalkneethling** [comment]
Yes, good call. I found this inside `sandstone.less` but, much better to use a standard attr if one exists.

**Comment by schalkneethling** [comment]
@alexgibson updated. Thanks for the suggestions.

File: less/tabzilla.less [hunk 2]
```diff
@@ -31,3 +31,9 @@
         z-index: 1;
     }
 }
+
+html[dir="rtl"] {
```
Comment (alexgibson) [inline less/tabzilla.less]: Nit: use single quotes for consistency

**Comment by alexgibson** [comment]
Looks good other than the last small nit - care to update & squarsh?

**Comment by schalkneethling** [comment]
@alexgibson squarshed

**Pull request merged**